content/docs/sidebar.json

```
@@ -477,5 +477,17 @@
         "slug": "cml-with-npm"
       }
     ]
+  },
+  {
+    "label": "dvclive",
+    "slug": "dvclive",
+    "source": "dvclive/index.md",
+    "children": [
+      "usage",
+      {
+        "label": "dvclive with DVC",
```

Are we just waiting for the name to merge?

```suggestion
        "label": "Dvclive with DVC",
```